.storybook/preview.js

```diff
@@ -4,6 +4,7 @@ import "@solana/wallet-adapter-react-ui/styles.css";
 import "../src/styles/globals.css";
 import "../src/styles/antd.less";
 import * as NextImage from "next/image";
+import "@ungap/has-own";
```

Comment:
The reason we need this is because we use `object.hasOwn` which is an API that is not available in older node versions.

Chromatic uses a version that does not support this. This "@ungap/has-own" is a polyfill that is only used for Chromatic.

I'll swap this out for the `core-js` package which is more widely used.

Reply:
Nvm, we can delete it when the helper functions get deleted based on your comment below.